Source/WebCore/rendering/svg/legacy/LegacyRenderSVGContainer.cpp

```diff
@@ -253,7 +253,14 @@ bool LegacyRenderSVGContainer::nodeAtFloatPoint(const HitTestRequest& request, H
             return true;
     }
     
-    // Spec: Only graphical elements can be targeted by the mouse, period.
+    // pointer-events=boundingBox makes it possible for containers to be direct targets
```

Comment:
Nit: https://webkit.org/code-style-guidelines/#comments-sentences
```suggestion
    // pointer-events=bounding-box makes it possible for containers to be direct targets.
```

Reply:
Done.